config/templates/module/tsconfig.build.json

```diff
@@ -1,8 +1,7 @@
 {
 	"extends": "./tsconfig.json",
 	"compilerOptions": {
-		"outDir": "./dist/",
-		"composite": true
```

Comment:
from the docs i thought composite was needed for project references: https://www.typescriptlang.org/docs/handbook/project-references.html#composite